models/maillog.go

```diff
@@ -27,6 +25,10 @@ var MaxSendAttempts = 8
 // MailLog is exceeded.
 var ErrMaxSendAttempts = errors.New("max send attempts exceeded")
 
+// processAttachment is used to to keep track of which email attachments have templated values.
```

Comment:
I don't think we need this anymore, right?

Reply:
Oops, correct.